### PR TITLE
yggdrasil 0.5.7

### PR DESCRIPTION
**Pull request by khipp**
Created by `brew bump`

---

Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/y/yggdrasil.rb [hunk 1]
```diff
@@ -13,5 +13,11 @@
   pkg "yggdrasil-#{version}-macos-#{arch}.pkg"
 
   uninstall launchctl: "yggdrasil",
-            pkgutil:   "io.github.yggdrasil-network.pkg"
+            pkgutil:   "io.github.yggdrasil-network.pkg",
+            delete:    [
+              "/etc/yggdrasil.conf",
+              "/Library/Preferences/Yggdrasil",
```
Comment (bevanjkay) [inline Casks/y/yggdrasil.rb:19]: These look like they could be config files? Are they better in `zap`?

Comment (khipp) [reply Casks/y/yggdrasil.rb:19]: Unfortunately, those files won't be removed when added to the `zap` stanza:

```console
$ brew uninstall --cask --zap y/yggdrasil.rb
==> Implied `brew uninstall --cask yggdrasil`
==> Removing launchctl service yggdrasil
Password:
==> Uninstalling packages with sudo; the password may be necessary:
io.github.yggdrasil-network.pkg
==> Dispatching zap stanza
==> Trashing files:
/etc/yggdrasil.conf
/Library/Preferences/Yggdrasil
==> Using sudo to gain ownership of path '/etc/yggdrasil.conf'
==> Using sudo to gain ownership of path '/Library/Preferences/Yggdrasil'
Warning: The following files could not be trashed, please do so manually:
/etc/yggdrasil.conf
/Library/Preferences/Yggdrasil
==> Removing all staged versions of Cask 'yggdrasil'
```

Comment (bevanjkay) [reply Casks/y/yggdrasil.rb:19]: What if you use zap delete

Comment (khipp) [reply Casks/y/yggdrasil.rb]: Thanks, that did it!

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**